wagtail/migrations/0093_uploadedfile_for_content_type.py

```diff
@@ -0,0 +1,20 @@
+# Generated by Django 5.0.1 on 2024-01-31 06:52
+
+import django.db.models.deletion
+from django.db import migrations, models
+
+
+class Migration(migrations.Migration):
+
+    dependencies = [
+        ('contenttypes', '0002_remove_content_type_name'),
+        ('wagtailcore', '0092_uploadedfile'),
+    ]
+
+    operations = [
+        migrations.AddField(
+            model_name='uploadedfile',
+            name='for_content_type',
+            field=models.ForeignKey(null=True, on_delete=django.db.models.deletion.CASCADE, related_name='uploads', to='contenttypes.contenttype', verbose_name='for content type'),
+        ),
+    ]
```

Comment:
It's best to have one new migration for this PR, I will redo these migrations when merging.

Just deleted the files and re-ran makemigrations.